action.yml

```diff
@@ -29,6 +29,10 @@ inputs:
   main_branch:
     required: false
     description: name of main branch of the repo, to get and store sha of the head commit and track diff being deployed
+  job_status:
```

Comment:
Do we actually need to pass this in? I saw that we have access to `github.event` in the `post` script so I'm wondering why it's different with `job`.

Reply:
You can do

```js
import * as github from '@actions/github';

github.context.repo
```

But we were not able to find how to access either `event` or `job` 🤔

Reply:
unfortunately no job context is accessible in JS code - https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions, only in YAML. Only env context and github context.

Reply:
also we can use states (which are essentially just process.env vars with STATE_ prefix), but we need to know what happened to additional steps after our action.

Reply:
Okay, cool 👍🏼

I'm still wondering if there's a secret way to do it. Asked in #c2c-actions-support, we can always adjust in the future if there actually are other ways.